mmcv/utils/logging.py

```diff
@@ -38,16 +39,7 @@ def get_logger(name, log_file=None, log_level=logging.INFO, file_mode='w'):
         if name.startswith(logger_name):
             return logger
 
-    # handle duplicate logs to the console
-    # Starting in 1.8.0, PyTorch DDP attaches a StreamHandler <stderr> (NOTSET)
-    # to the root logger. As logger.propagate is True by default, this root
-    # level handler causes logging messages from rank>0 processes to
-    # unexpectedly show up on the console, creating much unwanted clutter.
-    # To fix this issue, we set the root logger's StreamHandler, if any, to log
-    # at the ERROR level.
-    for handler in logger.root.handlers:
-        if type(handler) is logging.StreamHandler:
-            handler.setLevel(logging.ERROR)
+    logger.propagate = False
```

Comment:
We need to add a comment.

Reply:
@zhouzaida 
Please, check is it will be ok comment before 42th string:

# Handling of duplicate console logs.
# PyTorch DDP attaches a StreamHandler <stderr> (NOTSET)
# to the root logger. As logger.propagate is True by default, this root
# level handler causes logging messages from rank>0 processes to
# unexpectedly show up on the console, creating much unwanted clutter.

Reply:
Append the sentence 'More details can be found at PR #1683.'

Reply:
Hi @sleep3r , is there any progress?

Reply:
@HAOCHENYE @zhouzaida 
Sorry guys for this delay. I was shocked a bit due to Russian invasion of Ukraine :(

I've updated the comment.

Reply:
But now, tests aren't passing...

<img width="704" alt="Снимок экрана 2022-05-11 в 23 53 36" src="https://user-images.githubusercontent.com/33665156/167945774-d82d5a2a-a5aa-4d89-a88d-dc2b8a8a4f05.png">